### PR TITLE
Macosx build support

### PR DESCRIPTION
**Pull request by lucarosellini**
build support for MacOS X using brew installed gcc-4.8 using the following commands:

`brew tap homebrew/versions`
`brew install gcc@48`

if blackbird crashes with the following command:

`what():  locale::facet::_S_create_c_locale name not valid`

try to change LC_ALL to C:

`export LC_ALL="C"`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,6 +18,13 @@ SQLITE3 = libsqlite3.a
 SQLITE3CLI = sqlite3
 SQLITE3LIBS :=
 
+ifeq ($(OSNAME),Darwin)
+override INC_DIR += -I /usr/local/opt/openssl/include
+override LIB_DIR += -L /usr/local/opt/openssl/lib
+CC       := gcc-4.8
+CXX      := c++-4.8
```
Comment (greatwolf) [inline Makefile]: Best to not hardcode compiler version to the command. This is guaranteed to break if someone else builds with a different version.

Comment (lucarosellini) [inline Makefile]: @greatwolf you're definitely right. I will update this PR and add MacOS-related build section in the README.

**Comment by butor** [comment]
What do we do with this PR? Thanks

**Comment by greatwolf** [comment]
Maybe have someone else with a MacOS X test it make sure the changes work?

**Comment by calveto** [comment]
I tested this on my Mac.  I followed the instructions in the readme, but when starting blackbird I ran into the error mentioned in the description of this PR.  After exporting LC_ALL="C" per the fix mentioned above, blackbird was able to start successfully.

I would considering adding to the readme, the section from the description in this PR about blackbird crashing and exporting LC_ALL="C".

**Comment by lucarosellini** [comment]
@calveto you're right, I forgot to update the README, I'll do it asap

**Pull request closed without merging**